logging/logging-processor/src/main/java/org/infinispan/logging/processor/report/XmlReportWriter.java

```diff
@@ -0,0 +1,93 @@
+package org.infinispan.logging.processor.report;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+import javax.lang.model.element.Element;
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+
+import org.infinispan.logging.annotations.Description;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * XmlReportWriter.
+ * @author Durgesh Anaokar
```

Comment:
There should be a space between the Javadoc body and `@` tags. If you configure your ide to use the eclipse formatter in `ide-settings/`, you should be able to automatically format all of the files you have updated.